config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
It depends on what you consider the functionality of the app. If you wanted the styleguide mounted in each project, it would kind of belong to the app. If we only want to use it in dev mode, it's fine having it only in the dummy app. I think it would be nice to just have to go to a page instead of having to boot up an app locally, or at least having that option.